dropwizard-core/src/main/java/io/dropwizard/setup/Environment.java

```diff
@@ -118,7 +118,8 @@ public Environment(String name,
         } catch (IllegalStateException ignored) {
```

Comment:
The file has increased in 1 lines.
<!-- lookout footnote separator -->
_Comment made by the analyzer Dummy._